src/shared/components/palette/case-link/components/unlink-cases/unlink-cases.component.ts

```diff
@@ -33,16 +33,20 @@ export class UnLinkCasesComponent implements OnInit {
 
   public getLinkedCases(): void {
     this.caseId = this.linkedCasesService.caseId;
-    this.casesService.getCaseViewV2(this.caseId).subscribe((caseView: CaseView) => {
-      const linkedCasesTab = caseView.tabs.find(tab => tab.id === UnLinkCasesComponent.LINKED_CASES_TAB_ID);
-      if (linkedCasesTab) {
-        const linkedCases = linkedCasesTab.fields[0].value;
-        linkedCases.forEach(linkedCase => {
-          this.casesToUnlink.push(linkedCase.caseReference);
-        });
-        this.initForm();
-      }
-    });
+    if (this.linkedCasesService.linkedCases.length > 0) {
+      this.linkedCases = this.linkedCasesService.linkedCases;
+      this.initForm();
+    } else {
+      this.casesService.getCaseViewV2(this.caseId).subscribe((caseView: CaseView) => {
```

Comment:
I don't know whether you have to call the getCaseViewv2 as you may have the data already when you extend abstructfieldcomponent and use the this.caseField.value?